homeassistant/components/coolmaster/services.yaml

```diff
@@ -0,0 +1,18 @@
+feed:
+  name: Feed
+  description: Provide ambient temperature hint.
```

Comment:
Question: can these strings be translated? I checked the translations subdirectory in another integration (`tado`, which I used as an example to write the service) and there was nothing for services.